t/core/ctx.t

```diff
@@ -1008,3 +1008,23 @@ GET /hello
 hello world
 --- error_log eval
 qr/request log: \{"route_id":"1","route_name":"my_route","service_id":"1","service_name":"my_service"\}/
+
+
+
+=== TEST 36: support dash in the args
```

Comment:
Could you add the test in t/core/ctx2.t? Note that we already set up request & no_error_log there:
https://github.com/apache/apisix/blob/e3e018315311e0ca201f17579f9106eb88a834d1/t/core/ctx2.t#L23-L37

Reply:
ok